src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -12,9 +12,19 @@
 // limitations under the License.
 
 use super::{EvalContext, Result, ScalarFunc};
+use coprocessor::codec::error::Error;
+use coprocessor::codec::mysql::{self, Time};
 use coprocessor::codec::Datum;
 use std::borrow::Cow;
 
+fn handle_invalid_time_error(ctx: &mut EvalContext, err: Error) -> Option<Error> {
```

Comment:
Could we define this function in `EvalContext`? like `ctx.handle_division_by_zero` did（https://github.com/tikv/tikv/blob/master/src/coprocessor/dag/expr/ctx.rs#L228）

Reply:
Done